ReadMe.md

```diff
@@ -85,7 +86,9 @@ English | Japanese | Kana | Notes
 Development | 開発 | かいはつ |
 Developer | 開発者 | かいはつしゃ |
 Development department | 開発部 | かいはつぶ |
+Research and development, R&D | 開発研究 | けんきゅうかいはつ
 Implementation | 実装 | じっそう |
+Design | 設計 | せっけい | Can be loosely used as architecture, also used in other branches of engineering
```

Comment:
If you can tell me more exactly what I should do, perhaps I would be able to fix the description?